site/content/internal/release-process.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Release Process
+date: 2017-11-07T14:28:35-05:00
+section: internal
+---
+
+# Release Cutting Process
+
+**Note: To cut a build you need access to matterbuild. Please ask Christopher Speller for access if you don't have it.**
```

Comment:
No need to go to Speller:

* Add yourself in the platform-private repo: https://github.com/mattermost/platform-private/commit/89f91d81bd4602f4708270c0ca7626da8fc45291
* Trigger the matterbuild job: https://build.mattermost.com/job/matterbuild/